packages/aila/src/core/Aila.liveWithOpenAI.test.ts

```diff
@@ -13,7 +15,21 @@ const runManually = process.env.RUN_LLM_TESTS === "true";
 
     beforeEach(() => {
       ailaInstance = new Aila({
-        document: { content: {} },
+        document: {
+          content: {},
+          schema: LessonPlanSchema,
```

Comment:
This is where we say what kind of document we want the Aila instance to deal with.